docs/support/ghc-version-support.md

```diff
@@ -76,12 +80,43 @@ For example, if 9.2 is the latest major version fully supported by HLS and used
 
 HLS will support all non-legacy major versions of GHC.
 
-### Minor versions
+#### Minor versions
 
 For the latest supported major GHC version we will support at least 2 minor versions.
 
 For the rest of the supported major GHC versions, we will support at least the latest minor version in Stackage LTS (so 1 minor version).
 
+### Extended policy
+
+This is the part of the policy that needs evaluation by a human and possibly followed
+by a discussion.
+
+#### Ecosystem factors
+
+To establish and apply the policy we also take into account:
+
+- Completeness: support includes all plugins and features
```

Comment:
This is already included above

Reply:
I don't understand this comment

Reply:
Included where?

Do you mean 

> Support status (see the support policy below for more details):
> - "initial support": support for this GHC version is underway, but it is not ready to be released yet
> - "basic support": this GHC version is currently actively supported, and all [tier 1 plugins](./plugin-support.md) work
> - "full support": this GHC version is currently actively supported, and most [tier 2 plugins](./plugin-support.md) work
> - "deprecated": this GHC version was supported in the past, but is now deprecated

?

Reply:
I rephrased this to make sure it is clear we are referring to the HLS support status.

Also, I made clear that we aim to list *all* factors.